hera_pspec/testing.py

```diff
@@ -505,7 +505,7 @@ def sky_noise_sim(data, beam, cov_amp=1000, cov_length_scale=10, constant_per_bl
         Tsys = np.sqrt(autos[(bl[0], bl[0], bl[2])] * autos[(bl[1], bl[1], bl[2])])
 
         # get raw thermal noise
-        n = hs.noise.sky_noise_jy(Tsys, freqs/1e9, lsts, OmegaP[bl[2]], inttime=int_time)
+        n = hs.noise.sky_noise_jy(lsts, freqs/1e9,  autovis=Tsys, omega_p=OmegaP[bl[2]], integration_time=int_time)
```

Comment:
Ah, so there's a problem here: `sky_noise_jy` needs `autovis` to be in Jy. Since this is being passed in units of K, there's an extra conversion that's happening under the hood. So I think you want to do `autovis=Tsys/J2K` instead. (See line 118 in `hera_sim/noise.py`.) Sorry I didn't catch this before--I had to look at more of the `sky_noise_sim` function to figure this out.

Reply:
Ah, even easier... just never convert to K in the first place :-)

Reply:
Tada! Much easier...